structopt-derive/src/parse.rs

```diff
@@ -134,25 +139,35 @@ impl Parse for StructOptAttr {
                     if parser_specs.len() == 1 {
                         Ok(Parse(name, parser_specs[0].clone()))
                     } else {
-                        span_error!(name.span(), "parse must have exactly one argument")
+                        abort!(name.span(), "parse must have exactly one argument")
                     }
                 }
 
-                "raw" => {
-                    match nested.parse::<LitBool>() {
-                        Ok(bool_token) => {
-                            let expr = ExprLit { attrs: vec![], lit: Lit::Bool(bool_token) };
-                            let expr = Expr::Lit(expr);
-                            Ok(MethodCall(name, vec![expr]))
-                        }
+                "raw" => match nested.parse::<LitBool>() {
+                    Ok(bool_token) => {
+                        let expr = ExprLit {
+                            attrs: vec![],
+                            lit: Lit::Bool(bool_token),
+                        };
+                        let expr = Expr::Lit(expr);
+                        Ok(MethodCall(name, vec![expr]))
+                    }
 
-                        Err(_) => span_error!(name.span(),
-                            "`#[structopt(raw(...))` attributes are removed in structopt 0.3, they are replaced with raw methods\nhelp: if you meant to call `clap::Arg::raw()` method you should use bool literal, like `raw(true)` or `raw(false)`")
+                    Err(_) => {
+                        abort!(name.span(),
+                            "`#[structopt(raw(...))` attributes are removed in structopt 0.3, \
+                            they are replaced with raw methods";
+                            help = "if you meant to call `clap::Arg::raw()` method \
+                                you should use bool literal, like `raw(true)` or `raw(false)`";
+                            note = "if you need to call some method from `clap::Arg/App` you \
+                                should use raw method";
```

Comment:
I think "raw method" is not explicit enough, but I don't have a better idea.

Reply:
Well, we could try to parse erroneous `name = "litstr"` expression and suggest "you should use `name(litstr)`". I would involve pretty complicated post-processing (probably extracted into an external fn) but I believe it would worth the candle, don't you think?

Reply:
I decided to put some effort to this message since it's not the first time someone came complaining about it being unclear.